grading/execute.cpp

```diff
@@ -1486,6 +1521,9 @@ int execute(const std::string &cmd,
         }
         else{
           logfile << "Child exited with status = " << WEXITSTATUS(status) << std::endl;
+          if (!override) {
+            OutputSignalDescriptiveErrorMessageToExecuteLogfile(WEXITSTATUS(status), logfile, get_std_errfile(cmd));
```

Comment:
Is it possible to obtain the error by `time_kill` or `memory_kill`, rather than analyzing the `stderr`?

Reply:
The `time_kill` and `memory_kill` variables are not always `true` when program runs out of time or memory.